### ISSUE TITLE
Update Cinnamon theme for 3.2.1 change

### ISSUE THREAD

**Issue by DemonWav** [fusion809/macOS-Arc-White#1]
Info can be found here https://github.com/linuxmint/Cinnamon/issues/5934.

**Comment by fusion809** [comment]
I saw it but I didn't quite get what I was meant to do. Could you summarize it for me?

**Comment by DemonWav** [comment]
I read the issue wrong, sorry. It seems we'll either have to wait for this PR to pulled, to fix (or maybe after it's pulled the old behavior will return?) https://github.com/linuxmint/Cinnamon/pull/5948

**Comment by DemonWav** [comment]
https://github.com/linuxmint/Cinnamon/pull/5948 was pulled, and I've updated to Cinnamon 3.2.7, so apparently it's supposed to be fixed now, but it's not. I guess the implementation is different now.
